code/datums/ai/basic_mobs/basic_subtrees/run_emote.dm

```diff
@@ -0,0 +1,33 @@
+/// Intermittently run an emote
+/datum/ai_planning_subtree/run_emote
+	var/emote_key = BB_EMOTE_KEY
```

Comment:
didnt we have this kinda thing already for clowns

Reply:
no